src/WinRPM.jl

```diff
@@ -468,6 +468,12 @@ function do_install(package::Package)
 
     cpio = splitext(joinpath(cache, escape(basename(path))))[1] * ".cpio"
```

Comment:
The filenames of the downloaded rpm packages are escaped (e.g. `noarch%2Fmingw64-libstdc%2B%2B6-8.2.0-2.1.noarch.rpm`), but the cpio files after extraction with 7z are unescaped (e.g. `mingw64-libstdc++6-8.2.0-2.1.noarch.cpio`).

I guess `escape` in l.469 should be changed to `unescape` since
```julia
julia> using URIParser
julia> escape("c++")
"c%2B%2B"
julia> unescape("c%2B%2B")
"c++"
```

Reply:
That makes sense to me, I wonder how it *escaped* us!

@jcheyns What do you think about it? Does it fix the bugs you encountered?

Reply:
Sure, that was the fix I had. But in the original discussion (#141) we came to the conclusion to but a safety net in there and first check for the existence of the file, as no one seems very confident about this archane package.
I have this mod ready and works locally for me, as discussed I also want to have a couple of tests.
need to find some time for that. I can check in the PR and add the test later.

Reply:
yes and no. I originally started this discussion there